lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -1689,7 +1689,9 @@ def test_export_val_data_with_internal(self):
         # Also verify the content of created transcript file.
         expected_transcript_content = File(open(expected_transcript_path)).read()
         transcript = get_video_transcript_data(video_id=self.descriptor.edx_video_id, language_code=language_code)
-        self.assertEqual(transcript['content'], expected_transcript_content)
+
+        content = transcript['content'].decode('utf-8') if six.PY3 else transcript['content']
```

Comment:
Why aren't we always decoding this content?  That is what we've been doing historically.  Are there times when the content shouldn't be in unicode?

Reply:
content received in python 3 is in `bytes` and string in python 2

Reply:
But string in python will has the decode function so doing the same thing in python 2 and three here shouldn't cause any issues.

Reply:
decoding the `content` in python 2 converts it to Unicode that fails the comparison with the expected content.

Reply:
So it sounds like in python 2 the content is returned as bytes but in python3 it's being returned as unicode?  It sounds like we should update the upstream function to always return bytes for this content.